src/Controller/RedirectListBuilder.php

```diff
@@ -33,6 +34,7 @@ public function buildRow(EntityInterface $redirect) {
       $row['redirect_redirect']['data'] = '';
     }
     $row['status_code']['data'] = $redirect->getStatusCode();
+    $row['creation']['data'] = format_date($redirect->getCreationDateTime(), 'short');
```

Comment:
`Drupal::service('date.formatter')->format($redirect->getCreationDateTime(), 'short')` is the Drupal 8 way of doing that. 

the function you use basically just does this:

     function format_date($timestamp, $type = 'medium', $format = '', $timezone = NULL, $langcode = NULL) {
        return \Drupal::service('date.formatter')->format($timestamp, $type, $format, $timezone, $langcode);
     }